src/Persistence/Migrations/20240104122127_AddClanLanguages.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Crpg.Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddClanLanguages : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterDatabase()
+                .Annotation("Npgsql:Enum:activity_log_type", "user_created,user_deleted,user_renamed,user_rewarded,item_bought,item_sold,item_broke,item_reforged,item_repaired,item_upgraded,character_created,character_deleted,character_rating_reset,character_respecialized,character_retired,character_rewarded,server_joined,chat_message_sent,team_hit,clan_armory_add_item,clan_armory_remove_item,clan_armory_return_item,clan_armory_borrow_item")
+                .Annotation("Npgsql:Enum:battle_fighter_application_status", "pending,declined,accepted")
+                .Annotation("Npgsql:Enum:battle_mercenary_application_status", "pending,declined,accepted")
+                .Annotation("Npgsql:Enum:battle_phase", "preparation,hiring,scheduled,live,end")
+                .Annotation("Npgsql:Enum:battle_side", "attacker,defender")
+                .Annotation("Npgsql:Enum:character_class", "peasant,infantry,shock_infantry,skirmisher,crossbowman,archer,cavalry,mounted_archer")
```

Comment:
Remove Annotation that are not linked to language

Reply:
++
I will review and edit automatic migrations in the future